components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -35,7 +33,7 @@ type Props = {
     modalButtonDefaultText?: string;
     showLinkAsButton: boolean;
     icon?: React.ReactNode;
-    warnMetricStatus?: Dictionary<WarnMetricStatus>;
```

Comment:
Is this correct? It looks like we're still using it as a dictionary in `ConfigurationAnnouncementBar`

Reply:
in `announcement_bar/index.ts` we get `warnMetricsStatus` with type ` Dictionary<WarnMetricStatus>` from `getWarnMetricsStatus`.
then we pass it to `AnnouncementBarController`and then from there to `ConfigurationAnnouncementBar`. in   `ConfigurationAnnouncementBar` we loop over `warnMetricsStatus` dictionary and pass `status` to `AnnouncementBar`. 

also after your comment, I checked the code and find out in the `WarnMetricAckModal` (the modal type we call in `AnnouncementBar`), the `warnMetricStatus` prop has type `any` which I guess is wrong and should be `WarnMetricStatus`.

Reply:
Good catch on the `any`. I guess I just got a bit confused reading the code. This looks good to me now.